examples/c/reflection/dser_type/src/main.c

```diff
@@ -0,0 +1,51 @@
+#include <dser_type.h>
+
+#include <stdio.h>
+
+
+typedef struct {
+    ecs_vec_t v_i32; // vector<ecs_i32_t>
+    ecs_vec_t vv_i32; // vector<vector<ecs_i32_t>>
+} MiscellaneousVectors;
```

Comment:
There already is an existing deserialize example: https://github.com/SanderMertens/flecs/blob/master/examples/cpp/reflection/basics_deserialize/src/main.cpp

Maybe this example should be called "builtin_vectors"?

Also, suggestion to just call this type `Vectors`.

Reply:
Fixed